src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -39,7 +39,7 @@
         "symfony/stopwatch": "^6.4|^7.0",
         "symfony/translation": "^6.4|^7.0",
         "symfony/uid": "^6.4|^7.0",
-        "symfony/validator": "^6.4|^7.0",
+        "symfony/validator": "^7.0",
```

Comment:
until now, we managed to not do this anywhere
are these changes to fix existing failing tests or to accommodate changes made in this PR?
this PR should only contain what is required for it

Reply:
You're right, all the classes needed in these components are already present in `symfony/validator ^6.4`. I've reverted these changes.